### PR TITLE
Checking if park position has reached when motors has stopped

### PR DESCRIPTION
**Pull request by pmneo**

### CODE REVIEW

File: indi-eqmod/eqmodbase.cpp [hunk 1]
```diff
@@ -1065,7 +1065,22 @@ bool EQMod::ReadScopeStatus()
         {
             if (!(mount->IsRARunning()) && !(mount->IsDERunning()))
             {
-                SetParked(true);
+                currentRAEncoder = mount->GetRAEncoder();
+                currentDEEncoder = mount->GetDEEncoder();
+                parkRAEncoder    = GetAxis1Park();
+                parkDEEncoder    = GetAxis2Park();
+                if( static_cast<int32_t>(parkRAEncoder - currentRAEncoder) != 0 || static_cast<int32_t>(parkDEEncoder - currentDEEncoder) != 0 ) {
```
Comment (knro) [inline indi-eqmod/eqmodbase.cpp:1072]: Use std::abs(a-b) > 0 || ...etc which is better

Comment (pmneo) [reply indi-eqmod/eqmodbase.cpp:1072]: Whats about 
```
if( parkRAEncoder != currentRAEncoder || parkDEEncoder != currentDEEncoder )
```
?

File: indi-eqmod/eqmodbase.cpp [hunk 2]
```diff
@@ -1069,7 +1069,7 @@ bool EQMod::ReadScopeStatus()
                 currentDEEncoder = mount->GetDEEncoder();
                 parkRAEncoder    = GetAxis1Park();
                 parkDEEncoder    = GetAxis2Park();
-                if( static_cast<int32_t>(parkRAEncoder - currentRAEncoder) != 0 || static_cast<int32_t>(parkDEEncoder - currentDEEncoder) != 0 ) {
+                if( parkRAEncoder != currentRAEncoder || parkDEEncoder != currentDEEncoder ) {
```
Comment (knro) [inline indi-eqmod/eqmodbase.cpp:1072]: How about we use std::abs(a - b) > threshold ? If the different is like 1 tick, should we really issue a slew command? there should be an acceptable minimal threshold. maybe 5 or 10 ticks

Comment (pmneo) [reply indi-eqmod/eqmodbase.cpp]: Done, but why are the builds failing in a class which was not change in my branch?

**Comment by pmneo** [comment]
So I can confirm, that my fix applied to night:

[2024-02-10T05:43:04.346 CET INFO ][           org.kde.kstars.indi] - EQMod Mount :  "[INFO] Motors while parking stopped, reparking mount: RA increment = 892955, DE increment = -1059060 "

CS

**Pull request merged**